first-aid.lic

```diff
@@ -10,39 +10,36 @@ class FirstAid
   def initialize
     settings = get_settings
     @chart_data = get_data('anatomy-charts').first_aid_charts
-    @textbook = settings.textbook
-    if @textbook
+    if settings.textbook
       @booktype = 'textbook'
       textbook_charts
     else
       @booktype = 'compendium'
       compendium_charts
     end
   end
-  
   def compendium_charts
     bput('get my compendium', 'You get')
     bput('look my compendium', '^The compendium lies open to the section on .* physiology')
     pause
     charts = reget(40).grep(/^   .+$/).map(&:strip)
```

Comment:
`charts` is now kind of an ambiguous name. What do you think about `charts_in_compendium`?